### ISSUE TITLE
"Labor" is deduced from "Erreger" even if "Erreger" is marked as unknown

### ISSUE THREAD

**Issue by GerberI** [SiLeBAT/mibi-portal-server#244]
In a straight forward manner the "Labor" should be only deduced if the "Erreger" is marked as valid. If not it could lead to a user thinking the Portal is doing some magic they do not understand, making the usage more uncomfortable.

![Image](https://github.com/user-attachments/assets/51349718-0a5b-4a87-951e-dc2ad5ce7e13)

**Comment by afalenski** [comment]
This is true but not a bad situation: The user can see that the pathogen (in this case some Bacillus) is generally analysed by NRLs. The pathogen cell is coloured in red and the user can use the dropdown menu to select a valid pathogen name.

Will not be done.
